src/es/components/atoms/breadcrumb/Breadcrumb.js

```diff
@@ -11,7 +11,7 @@ import { Shadow } from '../../prototypes/Shadow.js'
  */
 export default class Breadcrumb extends Shadow() {
   constructor (options = {}, ...args) {
-    super(Object.assign(options, { importMetaUrl: import.meta.url }), ...args)
+    super({ ...options, importMetaUrl: import.meta.url }, ...args)
```

Comment:
The options object is first spread using the ... operator before assigning import.meta.url. 
This approach avoids modifying the original options object.